pkg/kudoctl/util/repo/repo_framework.go

```diff
@@ -83,7 +76,7 @@ func (r *FrameworkRepository) GetPackage(bundleName string) (*PackageCRDs, error
 	}
 
 	// first try old package format
-	fvPackage, err := ReadTarballPackage(resp)
+	fvPackage, err := ReadTarGzPackage(resp)
```

Comment:
```suggestion
	fvPackage, err := parsePackage(resp)
```

?

Reply:
On one hand, I'm not happy with the names either, on the other `ReadTarGzPackage(r io.Reader)` expects a `tar.gz` stream so the name is accurate and sets the expectations. Also, it's alternative is called `ReadFileSystemPackage` which expects either local file or folder and both basically build the package interface.